apluslms_roman/settings.py

```diff
@@ -47,7 +47,6 @@ def get_config_path(cls, filename=None):
     def load(cls, path, allow_missing=False):
         return super().load(path, version=cls.version, allow_missing=allow_missing)
 
-
```

Comment:
This is probably here by accident?

Reply:
what exactly should be done about it? another commit where the line is added back or rebase it so that the commit isn't made..?

Reply:
You can rebase (fixup / squash) to remove the change.